### ISSUE TITLE
Images in the “My Projects” popup menu should have an empty alt tag

### ISSUE THREAD

**Issue by phacks** [theodo/falco#139]
Thanks to @AlbericTrancart for the report.

From https://a11yproject.com/checklist/#section-images

> Decorative images do not communicate information that is required to understanding the website's content. Historically they were used for flourishes and spacer gif images, but are less relevant for modern websites. Empty alt attributes are also sometimes known as null alt attributes.
